packages/block-library/src/navigation/edit/index.js

```diff
@@ -73,6 +73,29 @@ import AccessibleDescription from './accessible-description';
 import AccessibleMenuDescription from './accessible-menu-description';
 import { unlock } from '../../lock-unlock';
 
+function useResponsiveMenu( navRef ) {
+	const [ isResponsiveMenuOpen, setResponsiveMenuVisibility ] =
+		useState( false );
+
+	useEffect( () => {
+		if ( ! navRef.current ) {
+			return;
+		}
+
+		const htmlElement = navRef.current.closest( 'html' );
```

Comment:
Tiny nit: This appears to be traversing up from the current ref to get to `html`. For consistency with how the `view.js` approach handles it, would it be worth using `documentElement` instead? I.e. something like:

```
const htmlElement = navRef.current.ownerDocument.documentElement;
```

Reply:
Sure! Much better! https://github.com/WordPress/gutenberg/pull/66093/commits/efe0a9ea417413c76635ba1e0fda5c52ff551ec9